provision/main.tf

```diff
@@ -0,0 +1,25 @@
+variable "aws_region" {
+  default = "us-east-2"
+}
+
+provider "aws" {
+  profile = "interviews-provision"
+  region  = "${var.aws_region}"
+}
+
+# terraform {
```

Comment:
If this commented block is not needed, could you remove it?

Reply:
Yes, actually I'll remove the whole file, it is only for explanatory purposes.